diff -up sysprof-3.36.0/src/libsysprof-capture/sysprof-capture-types.h.omv~ sysprof-3.36.0/src/libsysprof-capture/sysprof-capture-types.h
--- sysprof-3.36.0/src/libsysprof-capture/sysprof-capture-types.h.omv~	2020-03-15 14:04:37.255091915 +0100
+++ sysprof-3.36.0/src/libsysprof-capture/sysprof-capture-types.h	2020-03-15 14:04:46.906018525 +0100
@@ -73,7 +73,7 @@ G_BEGIN_DECLS
 # define SYSPROF_ALIGNED_END(_N) __attribute__((aligned ((_N))))
 #endif
 
-#define SYSPROF_CAPTURE_ADDRESS_FORMAT "0x%016"G_GINT64_MODIFIER"x"
+#define SYSPROF_CAPTURE_ADDRESS_FORMAT "0x%016" G_GINT64_MODIFIER "x"
 
 #if GLIB_SIZEOF_VOID_P == 8
 # define SYSPROF_CAPTURE_JITMAP_MARK    G_GUINT64_CONSTANT(0xE000000000000000)
